### PR TITLE
Added the information related to Symfony framework

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: source/_posts/symfony.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+layout: "post"
+color: "FFF"
+background: "222"
+author: "Javier Eguiluz"
+author_twitter: "javiereguiluz"
+gravatar: "9dc22dfc54bd9c666c72ebcd73d2fdd7"
+sponsor: "SensioLabs"
+sponsor_url: "https://sensiolabs.com"
+sponsor_twitter: sensiolabs
+sponsor_image: "http://securepasswords.info/themes/dshafik/securepasswords.info/assets/images/engineyard.png"
```
Comment (xabbuh) [inline source/_posts/symfony.md]: Looks like a wrong filename here.

Comment (javiereguiluz) [inline source/_posts/symfony.md]: Fixed. Thanks!

File: source/_posts/symfony.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+`$user` is the object that represents the user and it must implement the
+`Symfony\Component\Security\Core\User\UserInterface` interface.
+
+### Verifying a Password
+
+Make use of the `security.password_encoder` service:
+
+```php
+$isValid = $this->container->get('security.password_encoder')
+    ->isPasswordValid($user, $plainPassword);
```
Comment (xabbuh) [inline source/_posts/symfony.md]: The `security.password_encoder` service was introduced in Symfony 2.6. You should probably also document the old way to encode and validate passwords.

File: source/_posts/symfony.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+sponsor_url: "https://sensiolabs.com"
+sponsor_twitter: sensiolabs
+sponsor_image: "http://securepasswords.info/themes/dshafik/securepasswords.info/assets/images/engineyard.png"
+---
+
+## Installation
+
+Symfony relies on PHP hashing algorithms to encode passwords. The recommended
+algorithm is called `bcrypt` and it's available in PHP 5.5+. If you are using
+an earlier PHP version, install the `password_compat` library in your projects
```
Comment (xabbuh) [inline source/_posts/symfony.md]: I would add a link to the library here (either its GitHub repository or the Packagist page).

File: source/_posts/symfony.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+sponsor: "SensioLabs"
+sponsor_url: "https://sensiolabs.com"
+sponsor_twitter: sensiolabs
+sponsor_image: "http://securepasswords.info/themes/dshafik/securepasswords.info/assets/images/engineyard.png"
+---
+
+## Installation
+
+Symfony relies on PHP hashing algorithms to encode passwords. The recommended
+algorithm is called `bcrypt` and it's available in PHP 5.5+. If you are using
```
Comment (xabbuh) [inline source/_posts/symfony.md]: What about linking to the PHP documentation here ([Aura](https://raw.githubusercontent.com/dshafik/securepasswords.info/master/source/_posts/aura.md) for example does it this way).

File: source/_posts/symfony.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    ->encodePassword($user, $plainPassword);
+```
+
+`$user` is the object that represents the user and it must implement the
+`Symfony\Component\Security\Core\User\UserInterface` interface.
+
+In **previous Symfony versions**, use the `security.encoder_factory` service:
+
+```php
+$encoder = $this->get('security.encoder_factory')->getEncoder($user);
```
Comment (stof) [inline source/_posts/symfony.md]: you should be consistent between ``$this->get()`` and ``$this->container->get()``

**Pull request merged**